docs/io/configuration/components/plasma.rst

```diff
@@ -32,6 +32,28 @@ version of ``macroatom`` in which only downward transitions are allowed (see :re
 Finally, ``w_epsilon`` describes the dilution factor to use to calculate :math:`J_\textrm{blue}` that are 0, which
 causes problems with the code (so :math:`J_\textrm{blue}` are set to a very small number).
 
+Continuum Interaction
+^^^^^^^^^^^^^^^^^^^^^
+
+NLTE
+^^^^
+
+.. code-block:: yaml
+
+    plasma:
+        link_t_rad_t_electron: 1.0
+        continuum_interaction:
+            species:
+                - H I
+                - H II
+                - He I
+                - He II 
+            enable_adiabatic_cooling: True
+
+This will add continuum interactions for all specified species. Setting :math:`T_\textrm{rad} = T_\textrm{electron}` through 
+``link_t_rad_t_electron: 1.0`` is recommended to enforce LTE (unless the simulation uses NLTE treatment). 
+``enable_adiabatic_cooling`` enables adiabatic cooling.
```

Comment:
@sonachitchyan / @chvogl  Could you maybe add some more context for the ``enable_adiabatic_cooling`` option and generally check that what I wrote here is not completely wrong? Any additional explanations are welcome!